08_VirtualFileSystem/03_TarFileSystem.md

```diff
@@ -33,14 +33,15 @@ As anticipated above, the header structure is a fixed size struct of 512 bytes.
 | 148 | 8 	| Checksum for header record |
 | 156 | 1 	| Type flag |
 | 157 | 100 | Name of linked file |
-| 57  | 6 	| UStar indicator, "ustar", then NULL |
+| 257  | 6 	| UStar indicator, "ustar", then NULL |
 | 263 | 2 	| UStar version, "00" (it is a string) |
 | 265 |	32 	| Owner user name |
 | 297 |	32 	| Owner group name |
 | 329 |	8 	| Device major number |
 | 337 |	8 	| Device minor number |
 | 345 |	155 | Filename prefix |
 
+The sum of all sizes, anyway is not 512 bytes, but 500, so the extra space is filled with zerosextra space is filled with _0s_.
```

Comment:
This could be reflowed to sound smoother: `While a tar file header occupies 512 bytes, currently only 500 bytes are defined. The extra space is filled with zeroes.`